config/routes.rb

```diff
@@ -5,10 +5,7 @@
   get 'new_member' => 'benefits_applications#new_member'
   post 'create_member' => 'benefits_applications#create_member'
   get 'validate_application' => 'benefits_applications#validate_application'
-
-  # get 'benefits_applications/new', as: :new_benefit_app
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
+  patch 'update_member/:id/update', to: 'benefits_applications#update_member', as: 'update_member'
+  get 'members/:id/edit', to: 'benefits_applications#edit_member', as: 'edit_member'
```

Comment:
We should look into making a naming convention (noting because this and the deletion PR have it in a bunch of different ways). It's probably not a big deal right now.

Reply:
@lkemperman-cfa suggested moving the controller actions specifically for members